datalad_ria/tests/fixtures.py

```diff
@@ -72,3 +72,11 @@ def ria_sshserver(ria_sshserver_setup, monkeypatch):
     with monkeypatch.context() as m:
         m.setenv("DATALAD_SSH_IDENTITYFILE", ria_sshserver_setup['SSH_SECKEY'])
         yield ria_baseurl, ria_sshserver_setup['LOCALPATH']
+
+
+@pytest.fixture(autouse=False, scope="function")
```

Comment:
Technically not necessary to be function-scope, could be session-scope (for now), but also fine as-is.

Reply:
thanks. I've kept it as is for now.